data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -25,6 +25,17 @@ do
     sudo podman inspect "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.inspect"
 done
 
+echo "Gathering rendered assets..."
+mkdir -p "${ARTIFACTS}/rendered-assets"
+cp -r /var/opt/openshift/ "${ARTIFACTS}/rendered-assets"
```

Comment:
can we start with `/opt/*-bootstrap` dirs which are actually the dynamic assets created by operators?

Reply:
> can we start with `/opt/*-bootstrap` dirs which are actually the dynamic assets created by operators?

Everything here looks like it ends up created in our cluster and could fail in the log.  I'd like to get them all.

Reply:
What's the decision here?